tests/network/vmi_multus.go

```diff
@@ -1022,6 +1022,59 @@ var _ = Describe("[Serial]SRIOV", func() {
 				expectInterfaceToExistByMac(virtClient, vmi, mac, hotplugSRIOVDeviceTimeout,
 					"SR-IOV VF is expected to exist in the guest after migration")
 			})
+
+			Context("when aborted", func() {
+				const migrationRunningTimeout = 180
+				const migrationCompleteTimeout = 180
+
+				const targetPodDisposalTimeout = 120
+
+				const vmiMemoryStressSize = 200
+				const vmiStressTimeout = 600
+
+				var migration *v1.VirtualMachineInstanceMigration
+				var migrationUID string
```

Comment:
Why this is needed? I see that it can just be extrcted from the VMIM.

Reply:
I will remove this and extract it at the tests spec